cellxgene_schema_cli/cellxgene_schema/cli.py

```diff
@@ -79,21 +79,23 @@ def remove_labels(input_file, output_file):
 
 
 @click.command(
-    name="convert",
-    short_help="Convert an h5ad from version 2.0.0 to version 3.0.0",
-    help="Convert an h5ad from version 2.0.0 to version 3.0.0. No validation will be performed on either"
+    name="migrate",
+    short_help="Convert an h5ad to the latest schema version.",
+    help="Convert an h5ad to the latest schema version. No validation will be performed on either"
```

Comment:
nit: if you agree, might be worth noting here that it only converts from penultimate minor version to latest schema minor version.